client/my-sites/plans/current-plan/jetpack-checklist/constants.ts

```diff
@@ -100,12 +100,14 @@ export const JETPACK_CHECKLIST_TASK_AKISMET: TaskUiDescription = {
 	title: translate( "We're automatically turning on spam filtering." ),
 	completedButtonText: translate( 'View spam stats' ),
 	completedTitle: translate( "We've automatically turned on spam filtering." ),
+	tourId: 'jetpackAkismet',
```

Comment:
Why are we adding this? There is no such tour, and the task is being marked as automatically completed.